x-pack/plugins/search_playground/public/components/sources_panel/create_index_callout.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiCallOut, EuiSpacer, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React, { useCallback, useMemo } from 'react';
+import { useKibana } from '../../hooks/use_kibana';
+import { CREATE_INDEX_LOCATOR_ID } from '../../../common';
+
+export const CreateIndexCallout: React.FC = () => {
+  const {
+    services: { application, share },
+  } = useKibana();
+  const createIndexLocator = useMemo(() => {
+    const locator = share.url.locators.get(CREATE_INDEX_LOCATOR_ID);
+
+    if (!locator) {
+      throw new Error(`Create index locator not registered`);
```

Comment:
We can't throw an error here, Serverless will not have this locator initially. we should instead set `createIndexLocator = null` if the locator is not found, and hide the create index button in that case. 

Given that I would actually change this to be the create index url, and do the `await createIndexLocator?.getUrl({});` in this handler. Then you can just show / hide the `Create an index` button is `createIndexUrl !== null`